pkg/controller/subscription/subscription_controller.go

```diff
@@ -369,6 +378,23 @@ func (r *ReconcileSubscription) doReconcile(instance *appv1.Subscription) error
 	}
 
 	if sub, ok := r.subscribers[subtype]; ok {
+		// If the app sub time window is blocked, un-subscribe the app sub
```

Comment:
not sure if the unsubscribe would delete the deployed resource tho... if so, is it what we want? 

I mean change from active to block, to me, means new objects won't be deployed, and old objects wouldn't be deleted...

again, if unsubscribe won't delete the old resource, then there's not a problem with this  logic

Reply:
As discussed,  when time window is blocked, any new update of the application. will be blocked. But the deployed resources should be remained.